frida-gum/src/lib.rs

```diff
@@ -79,7 +79,14 @@ use core::{
     feature = "backtrace",
     feature = "memory-access-monitor"
 )))]
-use alloc::string::String;
+use alloc::{string::String, sync::Arc};
+
+#[cfg(any(
```

Comment:
This should be the `std` feature you introduced in #156

Reply:
Done